firmware/config/boards/48way/connectors/main.yaml

```diff
@@ -0,0 +1,106 @@
+pins:
+    - id: E2
+      class: outputs
+      ts_name: Fan Relay (E2)
+    - id: E3
+      class: outputs
+      ts_name: Fuel Relay (E3)    
+    - id: E4
+      class: analog_inputs
+      ts_name: Launch Control (E4)      
+    - id: E5
+      class: outputs
+      ts_name: Tachometer (E5)
+    - id: [E6, E6]
+      class: [switch_inputs, event_inputs]
+      ts_name: Camshaft Input (E6)
+    - id: A0
+      class: analog_inputs
+      ts_name: Battery Reference Input (A0)
+    - id: A1
+      class: analog_inputs
+      ts_name: TPS Reference Input (A1)
+    - id: A2
+      class: analog_inputs
+      ts_name: CLT Reference Input (A2)
+    - id: A3
+      class: analog_inputs
```

Comment:
analog inputs should be using EFI_ADC_* notation not pin ID notation - please see how IAT and TPS is handled in other boards

Reply:
```
    - id: [A2, EFI_ADC_2]
      class: [switch_inputs, analog_inputs]
      ts_name: CLT Reference Input (A2)
```

such as this format @rusefillc  ?

Reply:
just use the simple form
```
  - pin: B3
    id: EFI_ADC_13
    class: analog_inputs
    function: IAT Sensor Input
    ts_name: B3 - IAT
    color: orange
```